tket-exts/tests/test_validate_exts.py

```diff
@@ -0,0 +1,36 @@
+from tket_exts import (
+    debug,
+    futures,
+    guppy,
+    opaque_bool,
+    qsystem,
+    qsystem_random,
+    qsystem_utils,
+    quantum,
+    result,
+    rotation,
+    wasm,
+)
+
+import pytest
+
+
+exts = [
+    debug,
+    futures,
+    guppy,
+    opaque_bool,
+    qsystem,
+    qsystem_random,
+    qsystem_utils,
+    quantum,
+    result,
+    rotation,
+    wasm,
+]
+
+
+@pytest.mark.parametrize("ext", exts)
+def test_validate_extension(ext):
+    ext()
```

Comment:
It would be nice to add an actual assert to the test, other than just not-raising -- is there a simple thing we can check about the result of `ext()`?

Reply:
Hmm, could do a json serialisation round trip? Or something like `len(ext().types) + len(ext().operations) > 1`